staff/sergio-luz/app-template/login.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>login</title>
+</head>
+<body>
+    <form action="action_page.php"> 
```

Comment:
you need to specify the method. is it a POST or a GET request?

Reply:
I think you don't need it:
Syntax
form action="URL"

https://www.w3schools.com/Tags/att_form_action.asp

Reply:
not necessary for this challenge